code/js/controllers/RadioGardenController.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+(function() {
+
+  var BaseController = require("BaseController");
+
+  new BaseController({
+    siteName: "Radio Garden",
+    playPause: "div[class*=\"PlayBar_controlsContainer\"]>div:nth-child(2)",
```

Comment:
Can this be `.PlayBar_controlsContainer>div:nth-child(2)` ? (and the rest below)

Reply:
No because the class looks like this: 
```html
<div class="PlayBar_controlsContainer__2eO6e">
```
So I'm guessing the `__2eO6e` part is going to change each time they push an update.

Reply:
Had a look at the site and wow. Everything on the page is like that. Good job finding what worked.